src/TikaOnDotNet.Core/TikaOnDotnet.Core.csproj

```diff
@@ -0,0 +1,24 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netcoreapp3.1</TargetFramework>
+    <!--<TargetFrameworks>net462;netcoreapp3.1;net6.0</TargetFrameworks>-->
+    <Version>2.4.1</Version>
+  </PropertyGroup>
+
+    <ItemGroup>
+        <PackageReference Include="IKVM" Version="8.2.1" />
+    </ItemGroup>
+
+    <ItemGroup>
+        <IkvmReference Include="..\tika-app-*.jar">
+            <AssemblyName>tika.core</AssemblyName>
+        </IkvmReference>
+        <ContentWithTargetPath Include="$(OutDir)tika.core.dll">
```

Comment:
`IkvmReference` functions exactly like `Reference`. As such, it does not copy the assemblies to the output directory by default. This is denoted by the `Private` metadata item.

So set `<Private>true</Private>` on the `IkvmReference` and it'll probably work.

Any metadata set on `IkvmReference` flows through to `Reference`.